src/ui/TemplateMessageItemBody/FallbackTemplateMessageItemBody.tsx

```diff
@@ -0,0 +1,59 @@
+import type { BaseMessage } from '@sendbird/chat/message';
+import React, { ReactElement, useContext } from 'react';
+import { LocalizationContext } from '../../lib/LocalizationContext';
+import { getClassName } from '../../utils';
+import Label, { LabelColors, LabelTypography } from '../Label';
+
+export interface FallbackTemplateMessageItemBodyProps {
+  className?: string | Array<string>;
+  message: BaseMessage;
+  isByMe?: boolean;
+}
+export function FallbackTemplateMessageItemBody({
+  className,
+  message,
+  isByMe,
+}: FallbackTemplateMessageItemBodyProps): ReactElement {
+  const { stringSet } = useContext(LocalizationContext);
+  const text = message['message'];
```

Comment:
Is a message that can include a template in extendedMessage always a `UserMessage`?

Reply:
no there is no restriction that is why i put
```
{
        text
          ? <>
            <Label
              type={LabelTypography.BODY_1}
              color={LabelColors.ONCONTENT_INVERSE_1}
            >
              {text}
            </Label>
          </>
          : <>
```

Reply:
got it